### DISCUSSION TITLE
Windows Defender detected the tmp file as Trojan....

### DISCUSSION THREAD

**Discussion by carstenlehmann** [jpochyla/psst#380] [General]
.tmpoMNzMy was detected as Trojan:Script/Wacatac.B!ml

![image](https://user-images.githubusercontent.com/55583074/227228283-211fdc04-8503-432c-9921-472f69a9494d.png)

could be just a random thing... so i just don't open an issue.

Song was https://open.spotify.com/track/7peTraj0xsRKUlUNd0UyC7
Music stopped playing and got the Defender warning...

**Comment by carstenlehmann** [comment]
Update.
I can reproduce this fault at 1:06
just listend until 1:06 and defender blocks the tmp file and the music stops...

**Comment by Insprill** [comment]
Very interesting... I don't think there's anything we can do about that, but you can [submit it to Microsoft](https://www.microsoft.com/en-us/wdsi/filesubmission) to hopefully avoid the issue in the future.
